samples/bot-batch-operations/csharp/assets/sample.json

```
@@ -1,15 +1,15 @@
 [
   {
-    "name": "officedev-microsoft-teams-samples-bot-conversation-csharp",
+    "name": "officedev-microsoft-teams-samples-bot-batch-operations-csharp",
     "source": "officeDev",
-    "title": "Conversation Bot using Bot Framework v4",
-    "shortDescription": "This sample app demonstrates use of different bot conversation events available in bot framework v4 for personal and teams scope.",
-    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation/csharp",
+    "title": "Batch Operations Bot using Bot Framework v4",
+    "shortDescription": "This sample app demonstrates the different batch operations available from Bot Framework v4 for use in personal and teams scope.",
+    "url": "https://github.com/southworks/Microsoft-Teams-Samples/tree/southworks/add/batch-operations-input-actions/samples/bot-batch-operations/csharp",
```

Leave this url pointing to the main branch in MS repo. it should be:
```suggestion
    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-batch-operations/csharp",
```